br/tests/_utils/make_tiflash_config

```diff
@@ -42,16 +38,9 @@ tidb_status_addr = "127.0.0.1:10080"
 config = "$TEST_DIR/tiflash-learner.toml"
 log-file = "$TEST_DIR/tiflash-proxy.log"
 
-[flash.flash_cluster]
-cluster_manager_path = "$PWD/bin/flash_cluster_manager"
-log = "$TEST_DIR/tiflash-manager.log"
-master_ttl = 60
-refresh_interval = 20
-update_rule_interval = 5
-
 [logger]
 count = 20
-level = "trace"
+level = "debug"
```

Comment:
How much is the change's impact to test execution time?

Reply:
It doesn't affect the execution time much because the queries on those testing are not involve too much data in TiFlash.
I think "trace" is too verbose for integration testing, turn it to "debug" is enough.